actuator_coefficient:
  ddtP10: # DDTP10 with reducer
    act2pos: 0.00019174 # 2PI/32769
    act2vel: 0.1047197551   # 2PI/60
    act2effort: 0.025  # 75A/750*2.5Nm/A = 0.025  2.5Nm/A是转矩常数
    effort2act: 40.0   # 1/act2effort
    pos2act: 100
    pos_offset: 5866
    max_out: 20000